tests/plugins/custom-config.e2e.test.tsx

```
@@ -1,7 +1,8 @@
 import { describe, expect, it, vi } from 'vitest';
 import { page } from 'vitest/browser';
-import type { Config, PluginSetupParams } from '@/index.ts';
+import type { Config, PluginConfigApi, PluginSetupParams } from '@/index.ts';
```

Biome (2.4.13)

[error] Several of these imports are unused.

(lint/correctness/noUnusedImports)